.github/workflows/contract-tests.yml

```diff
@@ -32,3 +32,9 @@ jobs:
         env:
           REPORT_GAS: true
         run: yarn run test
+      - name: Test deployment (hardhat)
+        run: yarn deploy --network hardhat --reset
+        env:
+          ETH_KEY: ''
+          HARDHAT_DAO_ENS_DOMAIN: 'dao.eth'
+          HARDHAT_PLUGIN_ENS_DOMAIN: 'plugin.eth'
```

Comment:
This should be either `dao-plugin.eth`  or  `plugin.dao.eth`

Reply:
this will only run on `hardhat` network.
but, amended.